bears/js/HappinessLintBear.py

```diff
@@ -0,0 +1,24 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='happiness',
+        output_format='regex',
+        output_regex=r'\s.+:(?P<line>\d+):(?P<column>\d+):(?P<message>.+)'
+        )
+class HappinessLintBear:
+    """
+    Checks the js files using ``happiness`` for semantic and syntax errors.
```

Comment:
also replace ``js`` with double backticks or ``.js`` or say JavaScript. also the ``the`` is redundant